build/release.mk

```diff
@@ -53,13 +65,26 @@ ifeq ($(UPLOAD_PACKAGES), true)
 	@bash $(CURDIR)/build/upload_artifacts_gh.sh $(REPO_FULL_NAME)
 endif
 	@echo "===> $(INTEGRATION) === [release/publish] publishing manifests"
-	@$(GORELEASER_BIN) build --config $(CURDIR)/.goreleaser.yml --skip-validate --snapshot --rm-dist
+	@$(GORELEASER_BIN) build --config $(CURDIR)/.goreleaser.yml --skip=validate --snapshot --clean
 
+.PHONY : release/publish-fips
+release/publish-fips:
+ifeq ($(UPLOAD_PACKAGES), true)
+	@echo "===> $(INTEGRATION) === [release/publish] publishing packages"
+	# REPO_FULL_NAME here is only necessary for forks. It can be removed when this is merged into the original repo
+	@bash $(CURDIR)/build/upload_artifacts_gh.sh $(REPO_FULL_NAME)
+endif
+	@echo "===> $(INTEGRATION) === [release/publish] publishing manifests"
+	@$(GORELEASER_BIN) build --config $(CURDIR)/.goreleaser-fips.yml --skip=validate --snapshot --clean
```

Comment:
what is this doing?

Reply:
Changed echo statement, so that it makes sense.